assets/css/index.scss

```diff
@@ -0,0 +1 @@
+@import 'swagger-ui/dist/swagger-ui';
```

Comment:
We should rename this folder to `scss` and  have our specific styles within in  `api-docs/main.scss`